apps/website/src/pages/admin/unauthorized.tsx

```diff
@@ -0,0 +1,28 @@
+import type { NextPage } from "next";
+
+import Meta from "@/components/content/Meta";
+import Section from "@/components/content/Section";
+import Heading from "@/components/content/Heading";
+
+const UserUnauthorizedPage: NextPage<SigninPageProps> = () => {
```

Comment:
The props type is not needed.

```suggestion
const UserUnauthorizedPage: NextPage = () => {
```